restful_api/tests/factories.py

```diff
@@ -25,11 +25,22 @@ class EmployeeFactory(factory.Factory):
     id = factory.Sequence(lambda n: "%d" % n)
     name = factory.Sequence(lambda n: "employee %d" % n)
     user_type = factory.Sequence(lambda n: "user type %d" % n)
-    
     class Meta:
         model = Employee
 
+class OfficialEnrollFactory(factory.Factory):
+    eng_id = factory.Sequence(lambda n: 1)
+    course_id = factory.Sequence(lambda n: "%d" % n)
+    # start_date = factory.Sequence(lambda n: "%d" % n)
+    # end_date = factory.Sequence(lambda n: "%d" % n)
```

Comment:
Did you mean to comment out?

Reply:
Comment out for now, might test it in the future!